glean-core/csharp/Glean/Glean.cs

```diff
@@ -5,7 +5,11 @@
 using Mozilla.Glean.FFI;
 using static Mozilla.Glean.GleanMetrics.GleanInternalMetricsOuter;
 using System;
+using System.Runtime.CompilerServices;
+using Mozilla.Glean.Private;
+using System.Collections.Generic;
 
+[assembly: InternalsVisibleTo("GleanTests")]
```

Comment:
You can remove this: we now have a project level statement that enables it (rebase on latest `main`):